src/rules/declaration-empty-line-before/__tests__/index.js

```diff
@@ -191,6 +191,37 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { ignore: ["after-declaration"] } ],
+
+  accept: [ {
+    code: "\ntop: 15px; top: 15px;\n",
+  }, {
+    code: "\r\ntop: 15px; top: 15px;\r\n",
+  }, {
+    code: "\ntop: 15px;\ntop: 15px;\n",
+  }, {
+    code: "\r\ntop: 15px;\r\ntop: 15px;\r\n",
+  }, {
+    code: "\ntop: 15px;\n\ntop: 15px;\n",
+  }, {
+    code: "\r\ntop: 15px;\r\n\r\ntop: 15px;\r\n",
```

Comment:
Please put all these inside rules to better approximate real CSS. Or is there a specific reason you don't have them in rules?